app/styles/main.css

```diff
@@ -197,3 +197,11 @@ body {
   text-overflow: ellipsis;
   margin-bottom: 20px;
 }
+
+label,input {
```

Comment:
Lets move this after 
 body {
}

and add spacing between label and input like:

label, input {

}